src/client/app/audit/audit.module.ts

```diff
@@ -5,15 +5,17 @@ import { NgModule } from '@angular/core';
 import { Ng2BootstrapModule } from 'ng2-bootstrap';
 import { BootstrapModalModule, Modal } from 'angular2-modal/plugins/bootstrap';
 
+import { UtilModule } from 'app/shared';
+import { UserService } from 'app/admin/user';
+
 import { AuditViewChangeModal, AuditViewDetailModal } from './audit-view-change.component';
 import { AuditService } from './audit.service';
 import { AuditComponent } from './audit.component';
 import {
-	AuditObjectComponent, UrlAudit,
-	DefaultAudit, ExportAudit
+AuditObjectComponent, UrlAudit,
```

Comment:
Probably happened when I was reordering lines, IntelliJ likes to remove the indents.